lib/jsender/rack.rb

```diff
@@ -0,0 +1,44 @@
+module Jsender
+  module Rack
```

Comment:
I would add an extra namespace/naming to indicate the purpose of the module to consumers.
```
module Jsender
  module Helpers
   module Rack
```
or 
```
module Jsender
 module RackHelper
```

Reply:
Discussed on first discussion